Xamarin.PropertyEditing.Windows/Themes/WinThemeManager.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Reflection;
+using System.Resources;
+using System.Windows;
+using System.Windows.Baml2006;
+using System.Xaml;
+
+namespace Xamarin.PropertyEditing.Themes
+{
+
+	public class WinThemeManager : BaseThemeManager
+	{
+		Dictionary<string, ResourceDictionary> themeList;
+
+		public WinThemeManager ()
+		{
+			themeList = new Dictionary<string, ResourceDictionary> ();
+
+			var asm = Assembly.GetExecutingAssembly ();
```

Comment:
We can just hardcode a list of the themes and use `ResourceDictionary.Source` with a pack URI to import them instead of all this.

Reply:
While I could find a way to change ResourceDictionary.Source in Application.Current.Resources.MergedDictionaries, however I could not find a way to change the ResourceDictionary.Source inside the ExecutingAssembly. If you have an easy way to do that, great as it would simplify things greatly. I'll ping you on Monday about it.

Reply:
You don't have to change the source, you remove the current theme's merged dictionary and add the other one.